docresources/SyncBot/syncbot-readme.md

```diff
@@ -6,7 +6,11 @@ A little Bot, using the ProcBot framework, that links communication services
 
 ### How do I set up my accounts?
 
-* Ensure your Front, Discourse & Flowdock usernames match
+* Try:
```

Comment:
This is not useful for anyone apart from engineers who understand the 'try/catch' paradigm. I would not expect any of our staff who haven't written any code in a relevant language to understand why it's like this. Needs to be altered to something in an obvious English sentence.